Cargo.toml

```diff
@@ -5,3 +5,4 @@ authors = ["Samuel Marks <@SamuelMarks>"]
 edition = "2018"
 
 [dependencies]
+raft = { git = "https://github.com/pingcap/raft-rs" }
```

Comment:
It's releases are pretty frequent, just use one: https://github.com/pingcap/raft-rs/releases

[`cargo add`](https://github.com/killercup/cargo-edit) `raft-rs`